tasks/sapnote/0941735.yml

```diff
@@ -0,0 +1,15 @@
+---
+# tasks file for sap-preconfigure: configuration
+
+# Configure tmpfs
+- block:
+  - debug:
+      msg: "SAP note 0941735: Configure tmpfs;
+      memtotal_mb = {{ ansible_memtotal_mb }};
+      swaptotal_mb = {{ ansible_swaptotal_mb }};
+      sap_preconfigure_size_of_tmpfs_gb = {{ sap_preconfigure_size_of_tmpfs_gb }}"
+
+  - import_tasks: ../RedHat/generic/configure-tmpfs.yml
+  when: sap_preconfigure_config_all|d(true) or sap_preconfigure_0941735|d(false)
```

Comment:
indentation here

Reply:
oops ;-)

Reply:
I figured that the indentation was correct because the when is related to the block at the beginning of the file. I will add one more empty line above, for clarity.